### DISCUSSION TITLE
User-Profile in Front-End: missing redirect

### DISCUSSION THREAD

**Discussion by crimle** [joomla/joomla-cms#46999] [2. Cold Features]
### Steps to reproduce the issue
- Create a menu item of the type «Users > Edit User Profile»
- Log-in to the Front-End 
- Open the previously created menu item «User profile»
EITHER:
- Click the button [Cancel]
- Nothing happens
OR:
- Change whatever field of the user-profile you want, e.g. «Name» or «Password»
- Click the button [Save]
- Message «Profile saved.» appears but nothing else happens

### Expected result
In either case, one would expect the dialogue to be closed (similar to [Cancel] or [Save] after having edited articles). I would like to suggest a redirection to the previous page after both clicking [Cancel] and clicking [Save]. 

### Actual result
The «User profile» remains open. In case you have clicked [Cancel] you have the feeling that nothing happens. 

### System information (as much as possible)
Joomla 4.3.2

**Comment by obuisard** [comment]
See #38450 as similar issue.
